### PR TITLE
HMS-3240: Wizard: Proactive assistance - release lifecycle

### PR DESCRIPTION
**Pull request by regexowl**
This adds a lifecycle charts to the Wizard when a RHEL release is selected.

### CODE REVIEW

**Comment by briansmith0** [comment]
This looks awesome!  Would it be possible to provide links somewhere on this page where people can find the definition of "Full Support" and "Maintenance Support":

https://access.redhat.com/support/policy/updates/errata#Full_Support_Phase

https://access.redhat.com/support/policy/updates/errata#Maintenance_Support_2_Phase

**Comment by kelseamann** [comment]
Hey Klara if you see this, I just wanted to say it looks great! The line on the chart combined with the subheaders are *chefs kiss* this is an awesome improvement!

**Comment by mmenestr** [comment]
Going to add the link to documentation underneath the chart so it's a progressive flow of "Show me more info, oh that's not enough, let me click on more documentation" :) 

image

**Comment by regexowl** [comment]
I've added the link to docs to the expandable

image

File: src/test/jest.setup.js [hunk 1]
```diff
@@ -3,6 +3,12 @@ import failOnConsole from 'jest-fail-on-console';
 
 import { server } from './mocks/server';
 
+global.ResizeObserver = jest.fn().mockImplementation(() => ({
```
[the remark this answers is not in the document]
Comment (regexowl) [reply src/test/jest.setup.js:10]: Good point, the comment is added.

**Review by lucasgarfield — APPROVED** [review]
This is great! I can't believe how quickly you delivered this! :heart:

**Pull request merged**